tools/confix/cmd/home.go

```diff
@@ -2,21 +2,41 @@ package cmd
 
 import (
 	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/config"
 	"github.com/spf13/cobra"
+	"os"
+	"path/filepath"
 )
 
 func HomeCommand() *cobra.Command {
 	return &cobra.Command{
-		Use:   "home",
-		Short: "Outputs the string being used as the home path. No home directory is set when using the tool standalone.",
-		Args:  cobra.NoArgs,
-		Run: func(cmd *cobra.Command, args []string) {
+		Use:   "home [new-home]",
+		Short: "Read/Change the home directory. No home directory is set when using the tool standalone.",
+		Long:  "Read/Change the home directory. Given no argument, it outputs the string being used as the home path. Given one argument, the new home directory is set. No home directory is set when using the tool standalone.",
+		Args:  cobra.RangeArgs(0, 1),
+		RunE: func(cmd *cobra.Command, args []string) error {
 			clientCtx := client.GetClientContextFromCmd(cmd)
 			if clientCtx.HomeDir == "" {
 				cmd.Println("No home directory set.")
 			} else {
-				cmd.Println(clientCtx.HomeDir)
+				switch len(args) {
+				case 0: // if no arguments are given, print the home directory
+					cmd.Println(clientCtx.HomeDir)
+				case 1: // if one argument is given, set the home directory
+					newHome := args[0]
+					newConfigDir := filepath.Join(newHome, "config")
+					newConfigPath := filepath.Join(newConfigDir, "config.toml")
```

Comment:
Nit: Can we use one of the variable defined in confix? Like AppConfig? This avoids in confix to hardcode a name at multiple places.

Reply:
sure, I would agree that this is more elegant -- just a heads up, that there are a bunch of occurences of `config.toml` being hardcoded throughout the code base where it would probably make sense to replace it everywhere. That would make more sense on a dedicated PR though to keep concerns separated.